setup.sh

```diff
@@ -28,6 +27,10 @@ GRANT ALL PRIVILEGES ON \`patchdemo\_%\`.* TO 'patchdemo'@'localhost';
 composer update --no-dev
 npm install --production
 
+# setup daily cron job to deduplicate files
+BASEDIR=$(dirname "$0")
+sudo ln -s $BASEDIR/deduplicate.sh /etc/cron.daily/
```

Comment:
I didn't know jobs can be scheduled this way, neat.

Reply:
Neither did I until now.